modules/capture-admin-service-api/src/main/java/org/opencastproject/capture/admin/api/CaptureAgentStateService.java

```diff
@@ -93,6 +95,13 @@ public interface CaptureAgentStateService {
    */
   Map<String, Agent> getKnownAgents();
 
+  /**
+   * Returns the list of known agents that a given user and organization is authorized to schedule
+   *
+   * @return A {@link java.util.Map} of name-agent pairs.
+   */
+  Map<String, Agent> getKnownAgents(User u, Organization o);
```

Comment:
Why this new method if you run this in `runAs` anyway:

```java
SecurityUtil.runAs(securityService, org, systemAdminUser, () -> {
  …
}
```

Reply:
Good point, unnecessary changes.  Will revert.